rust/processor/migrations/2023-09-01-231248_events_v2/up.sql

```diff
@@ -0,0 +1,5 @@
+-- Your SQL goes here
+ALTER TABLE events DROP CONSTRAINT events_pkey;
+ALTER TABLE events DROP CONSTRAINT IF EXISTS fk_transaction_versions;
+ALTER TABLE events
+ADD CONSTRAINT events_pkey PRIMARY KEY (transaction_version, event_index);
```

Comment:
was this already done in your last PR? i think you need to rebase on main

Reply:
Yes